op-chain-ops/genesis/layer_one.go

```diff
@@ -65,12 +67,35 @@ func BuildL1DeveloperGenesis(config *DeployConfig) (*core.Genesis, error) {
 	if err != nil {
 		return nil, err
 	}
+	sysCfgABI, err := bindings.SystemConfigMetaData.GetAbi()
+	if err != nil {
+		return nil, err
+	}
+	data, err := sysCfgABI.Pack(
+		"initialize",
+		config.SystemConfigOwner,
```

Comment:
It looks like this value could be `address(0)`

Reply:
yep, updated the deploy configs in hardhat, that should fix the devnet. We'll also have to add it to hive setup.